README.md

```diff
@@ -1 +1,50 @@
-# earthly-mode
\ No newline at end of file
+# earthly-mode
+
+Major mode for editing Earthly Earthfile (https://docs.earthly.dev/docs/earthfile)
+
+## Current features
+
+* Syntax highlighting support.
+* Toggle comment.
+
+## Installation Notes
+
+### use-package
+
+If you're using `use-package`, you can do:
+
+```elisp
+(use-package
+  :ensure t)
+```
+
+### Manual
+
+Add this major mode to `load-path` using `add-to-list`:
+
+```elisp
+(add-to-list 'load-path "path/to/major/mode")
+```
+
+## Highlighting .earth file
+
+Currently, this major mode will highlight only `Earthfile` file. If you want
+to highligh `.earth` file extension (in https://github.com/earthly/earthly for example),
+put this configuration to your `init.el` or `.emacs` file:
+
+```elisp
+(add-to-list 'auto-mode-alist '("\\.earth\\'" . earthly-mode))
+```
+
+Or if you use `use-package`, just add:
+
+```elisp
+(use-package earthly-mode
+  :ensure t
+  :mode (("\\.earth\\'" . earthly-mode)
+         ("\\Earthfile\\'" . earthly-mode)))
+```
+
```

Comment:
This is no longer necessary, correct?

Reply:
You're right. It's remove in https://github.com/earthly/earthly-mode/pull/1/commits/dcbdeb306b5fe9b2fa231b22051d519470cd5f96.